src/components/Community/data.json

```diff
@@ -1,8 +1,8 @@
 {
   "hero": {
-    "pictureDesktop": "/img/community/banner.png",
-    "pictureMobile": "/img/community/banner-mobile.png",
-    "url": "https://www.mlprague.com/#schedule-saturday"
+    "pictureDesktop": "/img/community/header_mailing_list_skinny.png",
```

Comment:
@rogermparent when you start reconciling with your new PR - let's rename those to default-banner.png and default-mobile-banner.png ?

Reply:
Sounds good!